action.yml

```diff
@@ -3,6 +3,15 @@ description: 'Check if there are any unresolved merge conflicts'
 branding:
   icon: 'git-merge'
   color: 'gray-dark'
+inputs:
+  exclude_dir:
+    description: "Glob of directories to exclude from check"
+    required: false
+    default: 'node_modules,vendor'
+  excludes:
+    description: "Glob of files to exclude from check, supports wildcards"
+    required: false
+    default: ''
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  image: 'Dockerfile'
```

Comment:
Missing newline on this file, else we are good to go 👍

Reply:
Ah thats why that line was listed in the diff